src/reverse-proxy.c

```diff
@@ -166,7 +166,8 @@ char *reverse_rewrite_url (struct conn_s *connptr, hashmap_t hashofheaders,
                 return NULL;
         }
 
-        log_message (LOG_CONN, "Rewriting URL: %s -> %s", url, rewrite_url);
+        log_message (LOG_CONN, "Rewriting URL: %s -> %s", url, 
+			rewrite_url ? rewrite_url : "No mapping found");
```

Comment:
The indentation is done with tabs here. Should be spaces to be consistent with the rest of the code.
And should be aligned with the opening bracket.